app_spec/cluster_test/default.nix

```diff
@@ -1,16 +1,25 @@
 { pkgs }:
 let
-  name = "hc-app-spec-cluster-test";
+  name-cluster = "hc-app-spec-cluster-test";
 
-  script = pkgs.writeShellScriptBin name
+  script-cluster = pkgs.writeShellScriptBin name-cluster
   ''
   hc-cli-install
   hc-conductor-install
    ( cd hc_cluster_test && npm install && ./node_modules/.bin/tsc)
    ( cd app_spec && mkdir -p dist && hc package --output dist/app_spec.dna.json --strip-meta )
    ( EMULATION_HOLOCHAIN_BIN_PATH=./.cargo/bin/holochain node ./app_spec/cluster_test/index.js 2)
   '';
+
+  name-stress = "hc-stress-test-sim1h";
```

Comment:
Should this rather be in `stress-test/default.nix`?

Reply:
Yeah good call.